packages/cli/src/baseCommand.js

```diff
@@ -0,0 +1,58 @@
+const { Command } = require('@oclif/command');
+const { isEmpty } = require('lodash');
+const { cli: ux } = require('cli-ux');
+
+const { readCredentials, checkCredentials } = require('./utils/api');
+
+class ZapierBaseCommand extends Command {
+  _version() {
+    return 'blah blah ok'; // this should overwrite the version string, but doesn't?
+  }
+
+  log(message) {
+    if (!['json', 'raw'].includes(this.flags.format)) {
+      super.log(message || '');
+    }
+  }
+
+  get prompt() {
```

Comment:
we'll also get to use this instead of our own user-input, which is nice. I don't think we have any more complex inputs, but there are libraries (namely [Inquirer](https://github.com/SBoudrias/Inquirer.js/)) if we want to do checkboxes, etc.